djangocms_versioning_filer/helpers.py

```diff
@@ -95,10 +93,12 @@ def filename_exists(request, folder_id=None):
             upload = list(request.FILES.values())[0]
             filename = upload.name
         else:
-            filename = request.GET.get('qqfile', False) or request.GET.get('filename', False) or ''
+            filename = request.POST.get('file')
```

Comment:
What's the reason behind changing this? i think the code above is hit normally for request.FILES

Reply:
The code in if gets hit when files are dragged into the dropzone instance and the else when a file is uploaded with the file upload method. The later will be a post request hence the reason I am checking in POST data for the filename.

I believe the request.GET was implemented because the filename was passed on as a get parameter in the initial test_filename_exists test and not part of the post data even though it is doing a post in the test

https://github.com/divio/djangocms-versioning-filer/blob/release/0.0.x/tests/test_filename_exists.py#L22

Reply:
Ok mate, to save any confusion in the future can you please add comments for this?

Reply:
Will do